courses/machine_learning/production_ml/3_streaming_data.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>I'd rename the lab files as something more descriptive like:</p><p>3a_streaming_data_training.ipynb</p><p>3b_streamng_data_inference.ipynb</p><p> </p><p>By the way, the kubeflow pipeline lab is numbered "3" so maybe these labs should be numbered "4" instead.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/612/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>OK. Yes. Have renamed. thanks for catching that. </p>